packages/fxa-auth-server/lib/senders/templates/subscriptionFailedPaymentsCancellation.html

```diff
@@ -0,0 +1,15 @@
+{{! Cancellation email has no icon }}
+{{#*inline "icon"}}
+{{/inline}}
+
+{{! This email has no title }}
+{{#*inline "title"}}
+{{/inline}}
```

Comment:
Nit: Maybe we should have a title here? The email looks a little odd without one, and the other cancellation email has a title ("Sorry to see you go").

<img width="300" alt="Screen Shot 2021-07-07 at 2 27 00 PM" src="https://user-images.githubusercontent.com/17437436/124810520-6f60bd80-df2f-11eb-8fad-3c61e78e2cc3.png">
<img width="300" alt="Screen Shot 2021-07-07 at 2 27 09 PM" src="https://user-images.githubusercontent.com/17437436/124810522-6ff95400-df2f-11eb-8b13-506489cdf63e.png">

Reply:
Sure.  Should that be a follow-up or should it block the issue?  I prefer it as a follow-up as we might end up waiting a while for the content.  (Unless you have a suggestion for the title?? 😀)

Reply:
Definitely not blocking. Up to you.

Reply:
Hm... "Your subscription has been cancelled"?